securedrop/journalist_templates/admin_add_user.html

```diff
@@ -11,10 +11,16 @@ <h1 id="add-user-heading">Add User</h1>
     <div class="container flex-end">
       <div>
         <label for="username">{{ gettext('Username') }}</label>
-        {{ form.username() }}
+        {% set aria_attributes = {} %}
+        {% if form.username.errors %}
+        {% set _dummy = aria_attributes.update({'aria-describedby': 'username-errors', 'aria-invalid': 'true'}) %}
```

Comment:
jinja has a `do` expression that lets you avoid the set _dummy: https://jinja.palletsprojects.com/en/3.0.x/templates/#expression-statement

If we end up using this pattern elsewhere I think it would be nice to enable that extension.

Reply:
I like this suggestion.  We only use this pattern in one other template, in the Source Interface, but I've gone ahead and tacked on this quick refactoring in d72e0dc724f8d85e099215f34ac8f1f2d4afe94c.